src/sum.js

```diff
@@ -0,0 +1,15 @@
+export default sum
+
+/**
+* Original Source: http://stackoverflow.com/questions/1230233/how-to-find-the-sum-of-an-array-of-numbers
+*
+* This method will sum up all the array elements, assuming the elements are all numbers
+*
+* @param {Array} arr - the arr to be summed up
+*
+*/
+
+function sum(arr) {
+  return arr.reduce(function(a, b) {
```

Comment:
@IggHub Here's what (I think) is the next step in this, and any other lib.

1. Checks pass (tests, coverage, etc)
2. Since @kentcdodds requested changes, he will get notified of your changes in response to his request and not has the opportunity to review that those changes fix the issue noted by his first request.
3. Wait :)
4. Reviewer will either accept changes (approving PR) and merge, or issue additional change requests. The former is more likely here.

The takeaway is that once you get everything "proper" and ready for merging, you are at the mercy of the reviewer's/maintainer's timeframe to do the approve/merge. On FOSS projects, this can take some time, or maybe even never happens.

In that case, as a dev, I've forked the repo, implemented my changes, and still waited for final closure in the upstream repo.